integration/fields.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package integration
```

Comment:
"github.com/u-root/u-root/pkg/sh/" might be a good package to place the functions of this file.